ts/src/base/Exchange.ts

```diff
@@ -1431,6 +1417,29 @@ export default class Exchange {
         return this.filterByLimit (result, limit, key);
     }
 
+    setSandboxMode (enabled) {
+        if (this.notNotValue (enabled)) {
```

Comment:
this is blanket function in php/py, only in JS it does `!!` reversion

Reply:
@ttodua we don't need this, I think we can treat `enabled` as an integer and we also add the type to the argument btw

Reply:
`integer` u sure? maybe it should accept bool, it's more expected for "enable/disable" flag other than number (even 0/1 being most common integers for enabling/disabling)

Reply:
@ttodua sorry, I mean `boolean` 😅